resources/default_contacts.json

```diff
@@ -204,5 +204,18 @@
         {
             "en": "http://test.etherplay.io"
         }
+    },
+      "REXmls":
+    {
+        "name":
+        {
+            "en": "REXmls"
+        },
+        "photo-path": "contacts://rexmls",
+        "dapp?": true,
+        "dapp-url":
+        {
+            "en": "http://http://rexmls.com/"
```

Comment:
I don't see any DApp running on http://rexmls.com/ 
I believe contacts should contain a DApp instead of just adding a link to the webpage, @jarradh opinions about this?